hardware/arduino/sam/libraries/HID/HID.cpp

```diff
@@ -68,7 +68,7 @@ int HID_GetDescriptor(int8_t t)
 		HIDDescriptorListNode* current = rootNode;
 		int total = 0;
```

Comment:
Is there a reason why you always use int in the Arduino programs? uint16_t would make life much easier in most cases.